core/blockchain.go

```diff
@@ -1469,8 +1469,10 @@ func (bc *BlockChain) reportBlock(block *types.Block, receipts types.Receipts, e
 	bc.addBadBlock(block)
 
 	var receiptString string
-	for _, receipt := range receipts {
-		receiptString += fmt.Sprintf("\t%v\n", receipt)
+	for i, receipt := range receipts {
+		receiptString += fmt.Sprintf("\t %d: cumGasUsed: %v gasUsed: %v cAddress %v status %v txHash %v logs %v bloom %x postState %x\n",
```

Comment:
Please use `totalGasUsed`. Cumulative abbreviated sounds bad.
Please use `contract` instead of cAddress.
I think `tx` instead of `txhash` is pretty obvious, same for `state` instead of `postState`.
You added colons after the first couple fields but not all.

Reply:
Done (but used `cumulative` instead of `totalGasUsed`)